src/fprime_gds/common/testing_fw/api.py

```diff
@@ -63,6 +66,13 @@ def __init__(self, pipeline, logpath=None, fsw_order=True):
         # Initialize the logger
         self.logger = TestLogger(logpath) if logpath is not None else None
 
+        # Copy dictionaries and binary file to output directory
+        base_dir = Path(self.pipeline.dictionary_path).parents[1]
+        for subdir in ['bin', 'dict']:
+            shutil.copytree(os.path.join(str(base_dir), subdir),
```

Comment:
We tend to use the Path `/` operator rather than `os.path.join`, could we use that here and below?

`basedir / subdir` should work?